src/Module/Photo.php

```diff
@@ -74,9 +75,7 @@ public static function init()
 		}
 
 		if ($photo === false) {
-			// not using System::httpExit() because we don't want html here.
-			header($_SERVER["SERVER_PROTOCOL"] . " 404 Not Found" , true, 404);
-			exit();
+			System::httpExit('404', 'Not Found');
```

Comment:
Why don't we use an exception too?

Reply:
This module output image data, so it doesn't make sense to output HTML. It's the only other place in the code where calling `System::httpExit()` is actually warranted.